src/tls13cert.rs

```diff
@@ -281,6 +283,7 @@ fn read_spki(cert: &Bytes, mut offset: usize) -> Result<Spki, Asn1Error> {
 /// certificate.
 ///
 /// Returns the start offset within the `cert` bytes and length of the key.
+#[pv_handwritten]
```

Comment:
These annotations need to become conditional on the configuration so that bertie can be used by clients who do not install `hax_lib_macros`